src/classes/UTIL_OrgTelemetry_TEST.cls

```diff
@@ -36,6 +36,7 @@
 @IsTest(IsParallel=false)
 private class UTIL_OrgTelemetry_TEST {
 
+    private static UTIL_FeatureManagement_TEST.Mock featureManagementMock = new UTIL_FeatureManagement_TEST.Mock();
```

Comment:
You may be run this in `parallel=true` mode now that you're mocking the feature parameter part. Worth trying if you haven't already.